networkx/algorithms/approximation/traveling_salesman.py

```diff
@@ -1249,7 +1249,7 @@ def threshold_accepting_tsp(
     Parameters
     ----------
     G : Graph
-        `G` should be a complete weighted undirected graph.
+        `G` should be a complete weighted undirected(or directed) graph.
```

Comment:
It's probably more clear just to remove some wording rather than add the parenthetical:

```suggestion
        `G` should be a complete weighted graph.
```

Reply:
Done @rossbar 

Thanks!